src/diet/index.ts

```diff
@@ -439,6 +447,15 @@ class DietPlanner<T> {
       helpers.push(this.whetStone);
     }
 
+    if (
+      this.aioli &&
+      itemType(menuItem.item) === "food" &&
+      this.mpa * menuItem.item.fullness >
+        Math.min(mallPrice($item`mini kiwi aioli`))
```

Comment:
this should not use hte mall price, it should use the menuItem aioli.price

Reply:
Should Seasoning, Whetstone, etc. be updated as well?